components/DarkModeSwitch.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import Toggle from 'react-toggle';
+import useDarkMode from 'use-dark-mode';
+import SunIcon from './SunIcon';
+import MoonIcon from './MoonIcon';
+
+const STORAGE_KEY = 'DARK_MODE';
+
+const DarkModeSwitch = () => {
+    // if (typeof window === 'undefined') {
+    //     // Prevent the rendering of this component until it is in the browser
+    //     return <></>;
+    // }
```

Comment:
This is not used because of the dynamic in footer right?

Reply:
That's correct. I have just taken this out and updated my branch.